src/pkg/services/m365/api/users.go

```diff
@@ -200,12 +200,12 @@ func (c Users) GetInfo(ctx context.Context, userID string) (*UserInfo, error) {
 
 		if graph.IsErrUserNotFound(err) {
 			logger.CtxErr(ctx, err).Error("user not found")
-			return nil, clues.Stack(graph.ErrResourceOwnerNotFound, err)
+			return nil, clues.Wrap(clues.Stack(graph.ErrResourceOwnerNotFound, err), "")
 		}
 
 		if !graph.IsErrExchangeMailFolderNotFound(err) {
 			logger.CtxErr(ctx, err).Error("getting user's mail folder")
-			return nil, err
+			return nil, clues.Wrap(err, "")
```

Comment:
Should use Stack if you don't need a message.
```suggestion
			return nil, clues.Stack(err)
```

Reply:
stack doesn't seem to be giving stack trace information which is what I really want